src/heads-up/views/app-meta.vue

```diff
@@ -2,22 +2,71 @@
   <div>
     <panel-section title="Properties">
       <properties-list>
-        <dt>Title</dt><dd>{{ head.title }}</dd>
-        <dt>Language</dt>
-        <dd>{{ head.lang }}</dd>
-        <dt>Charset</dt><dd>{{ charset }}</dd>
-        <dt>Viewport</dt><dd>{{ viewport }}</dd>
-        <template v-if="themeColor">
-          <dt>Theme color</dt>
-          <dd>
+        <properties-item
```

Comment:
This looks like a lot of repetition. Is it possible to first create an array of objects that then could be used in a v-for loop?